images/runtime/dotnetcore/5.0/base.buster.Dockerfile

```diff
@@ -57,3 +55,11 @@ RUN set -ex \
     && rm aspnetcore.tar.gz \
     && dotnet-sos install \
     && rm -rf ${BUILD_DIR}
+
+# Install ca-certificates from bullseye repository: https://github.com/NuGet/Announcements/issues/49
```

Comment:
because of this we also need to create new sets of runtime images for dotnet core 5.0. You need to create new base image tag and then change the corresponding base image tag for dotnet core runtime images